src/viewer.js

```diff
@@ -668,154 +850,144 @@ class VolumeImageViewer {
-     * frames (load tiles) via DICOMweb WADO-RS.
-     */
-    const rasterSource = new TileImage({
-      crossOrigin: 'Anonymous',
-      tileGrid: tileGrid,
-      projection: projection,
-      wrapX: false
-    });
-    rasterSource.setTileUrlFunction(tileUrlFunction);
-    rasterSource.setTileLoadFunction(tileLoadFunction);
+    return channel;
+  }
 
-    this[_imageLayer] = new TileLayer({
-      extent: extent,
-      source: rasterSource,
-      preload: 0,
-      projection: projection
-    });
+  /** Gets the channel metadata given an id
+   * @param {string} id of the channel (opticalPathIdentifier)
+   * @type {metadata[]} array with all the instances metadata of the channel
+   */
+   getOpticalPathMetadata(opticalPathID) {
+    const channel = this.getOpticalPath(opticalPathID)
+    return channel ? channel.metadata : null;
+  }
 
-    this[_drawingSource] = new VectorSource({
-      tileGrid: tileGrid,
-      projection: projection,
-      features: this[_features],
-      wrapX: false
-    });
+  /** Sets the channel visualization/presentation parameters given an id
+   * @param {string} id of the channel (opticalPathIdentifier)
+   * @param {number[]} color
+   * @param {number} opacity
+   * @param {number[]} thresholdValues
+   * @param {boolean} visible
+   */
+  setOpticalPathBlendingInformation(
+    opticalPathID, 
+    color,
+    opacity,
+    thresholdValues,
+    visible) {
+
+    const channel = this.getOpticalPath(opticalPathID)
+    if (channel === null) {
+      return null;
+    }
 
-    this[_drawingLayer] = new VectorLayer({
-      extent: extent,
-      source: this[_drawingSource],
-      projection: projection,
-      updateWhileAnimating: true,
-      updateWhileInteracting: true,
-    });
+    if (channel.setBlendingInformation(
+        color,
+        opacity,
+        thresholdValues,
+        visible)
+      ) { 
+      this[_map].render();
+    }
+  }
 
-    const view = new View({
-      center: getCenter(extent),
-      extent: extent,
-      projection: projection,
-      resolutions: resolutions,
-      rotation: rotation
-    });
+  /** Gets the channel visualization/presentation parameters given an id
+   * @param {string} opticalPathID of the channel
+   * @param {number[]} color
+   * @param {number} opacity
+   * @param {number[]} thresholdValues
+   * @param {boolean} visible
+   */
+  getOpticalPathBlendingInformation(opticalPathID) {
+    const channel = this.getOpticalPath(opticalPathID)
+    if (channel === null) {
+      return null;
+    }
 
-    this[_interactions] = {
-      draw: undefined,
-      select: undefined,
-      modify: undefined
-    };
+    return channel.getBlendingInformation();
+  }
 
-    this[_controls] = {
-      scale: new ScaleLine({
-        units: 'metric',
-        className: ''
-      })
+  /** Adds the channel to the OpenLayer Map given an id
+   * @param {string} opticalPathID of the channel
+   */
+   activateOpticalPath(opticalPathID) {
```

Comment:
I would suggest also adding methods `showOpticalPath()` and `hideOpticalPath()` to allow users to toggle channel visibility without adding or removing the corresponding layers to or from the map, respectively.